src/draw.c

```diff
@@ -622,6 +636,9 @@ static void render_content(cairo_t *c, struct colored_layout *cl, int width)
                 // icon position
                 if (settings.icon_position == ICON_LEFT) {
                         text_x = cairo_image_surface_get_width(cl->icon) + 2 * settings.h_padding;
```

Comment:
This should be only 1 times h_padding

Reply:
I didnt actually modify that bit. I think it's 2*h_padding to provide the gap between icon and text the old way. Changing it would break current setups

Reply:
Oh sorry I didn't look good enough. It's good like this :)